tools/win_installer/misc/depcheck.py

```diff
@@ -20,15 +20,34 @@
 from multiprocessing import Process, Queue
 
 import gi  # isort:skip
-gi.require_version("GIRepository", "2.0")  # isort:skip
+
+girepository_version = 3
+try:
+    gi.require_version("GIRepository", "3.0")  # isort:skip
+    girepository_version = 3
+except ValueError as e:
+    girepository_version = 2
+    gi.require_version("GIRepository", "2.0")  # isort:skip
+
 from gi.repository import GIRepository  # isort:skip
 
 
 def _get_shared_libraries(q, namespace, version):
     repo = GIRepository.Repository()
     try:
         repo.require(namespace, version, 0)
-        lib = repo.get_shared_library(namespace)
+        if girepository_version == 3:
+            ret = repo.get_shared_libraries(namespace)
+            if ret:
+                lib = ret[0]
+            else:
+                lib = None
+        elif girepository_version == 2:
+            lib = repo.get_shared_library(namespace)
```

Comment:
Maybe you can do the `lib.lower().split(",")` here:
```suggestion
            lib = repo.get_shared_library(namespace)
            if lib:
              lib = lib.split(","))
```

and rename lib as libs.